### ISSUE TITLE
p2p 20/10/02

### ISSUE THREAD

**Issue by adamclasic** [adamclasic/Todo-list#2]

**Comment by nyatindopatrick** [comment]
Good work 👍 though in your imports you don't need default there when using export default. Destructuring imports should only be done when having named exports. Also arrow functions are much cleaner than using the `function` keyword. Apart from those, the project looks great!

**Comment by rammazzoti2000** [comment]
I agree with Patrick 👍 , great design, the index is a bit big, but apart from that all is great.
Keep up the great work.

**Comment by dili021** [comment]
What Patrick said.
You can look at the whole module (file) as being an object. So when you're exporting multiple things it's like exporting multiple properties of an object - the variables and functions so it's like
```
//the module "object"
{
someFunc: function........
otherFunc: function........
someConst: "string"
etc
}
```
If it's a default export, meaning your exporting only 1 thing you just `export default nameOfThing;`

Then on the import side, if you're importing named exports you're now accesing them by destructuring that module "object"
`import { someFunc, someConst } from ...`
Otherwise if you're importing default, there's no "object" so you just `import nameOfThing from ...`
